lib/customers.js

```diff
@@ -723,15 +721,18 @@ function getCustomerById (id) {
     WHERE c.id = $2
   ) AS cl WHERE rn = 1`
   return db.oneOrNone(sql, [passableErrorCodes, id])
-    .then(customerData => {
-      return getEditedData(id)
-        .then(customerEditedData => selectLatestData(customerData, customerEditedData))
-    })
+    .then(assignCustomerData)
     .then(populateOverrideUsernames)
     .then(camelize)
     .then(it => ({ ...it, notes: (it.notes ?? []).map(camelize) }))
 }
 
+function assignCustomerData (customer) {
+  return getEditedData(customer.id)
+    .then(customerEditedData => selectLatestData(customer, customerEditedData))
+    .then(_.assign(customer))
```

Comment:
I don't see why this is necessary, selectLatestData should preserve all the data and only update the required fields.

Reply:
At the time, before I added the `_.assign()`, the data from the `edited_customer_data` table wasn't sent to the machine even though it was fetched from the DB. Everything started working after I added it, but looking again now, I also agree it doesn't look like it's doing anything... I'll check!

Reply:
Indeed, looks like the `_.assign()` is unnecessary.